app/views/layouts/application.html.erb

```diff
@@ -29,18 +30,14 @@
   </head>
 
   <body>
-    <div id="su-wrap"> <!-- #su-wrap start -->
-      <div id="su-content"> <!-- #su-content start -->
-        <div id="outer-container" class="container-fluid">
-          <%= render 'shared/top_navbar' %>
-          <section id="main-container" role="main" class="container pt-2 pb-3">
-            <%= breadcrumbs style: :bootstrap5 %>
-            <%= render 'shared/global_alert' %>
-            <%= render 'shared/flashes' %>
-            <%= yield %>
-          </section>
-        </div>
-      </div>
+    <div id="outer-container" class="container-fluid">
```

Comment:
Sorry, I wonder if this one is needed too? In other sites we don't have this.

Reply:
We are going to need some class I think. I tried to pull out https://github.com/sul-dlss/library_hours_rails/pull/585/files#diff-4b04ba88a773a7736d1e8a5bcb978cf7faa74c935b4389bcd9d962407c9a8149R1-R5

Reply:
Actually removing it totally fixes this. Getting rid of container fluid.